lib/utils/signupValidator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable newline-per-chained-call */
+const { validateName, validateEmail, validatePassword } = require('./validate');
+const { failureResponse } = require('../utils/messageHandler');
+const { BAD_REQUEST_CODE } = require('../../constants/index');
+
+// const BAD_REQUEST_CODE = 400;
+
+const formatErrorMessage = (msgArr) => {
+  const message = {};
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
`// eslint-disable-next-line no-restricted-syntax`
Why are you disabling this rule?